src/vdaf/prio3.rs

```diff
@@ -223,6 +223,165 @@ where
     pub fn verifier_len(&self) -> usize {
         self.typ.verifier_len()
     }
+
```

Comment:
This should be identical to the previous `shard()` modulo replacing calls to `Seed::generate()` with calls to `Seed::from_rand_source(rand_source)`.